libyul/backends/evm/EVMCodeTransform.h

```diff
@@ -39,6 +39,8 @@ class ErrorReporter;
 
 namespace solidity::yul
 {
+langutil::SourceLocation extractSourceLocationFromDebugData(std::shared_ptr<DebugData const> const& _debugData);
+
```

Comment:
You do not need to put these utility functions in the header. You can just create an anonymous namespace in the cpp file.

Reply:
Done